package.json

```diff
@@ -25,10 +25,10 @@
     "query"
   ],
   "devDependencies": {
-    "jstestr": ">=0.4",
-    "pegjs": "~0.7.0",
     "commonjs-everywhere": "~0.9.4",
-    "esprima": "~1.1.1"
+    "esprima": "^4.0.0",
```

Comment:
I upgraded the `esprima` devDependency because `esprima@1.1.1` can't parse ES6 syntax. This isn't strictly necessary since the AST is hardcoded in the fixture file, but I thought it would be confusing to have a code sample in the fixture file that doesn't work with the installed version of esprima.